README.md

```diff
@@ -0,0 +1,137 @@
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
+
+"To-Do List" is a simple website that displays a list of activities and allows you to add and remove activities from that list. By building this application, we learned how to manage data using JavaScript. Thanks to that your website will be more interactive.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- JavaScript
+- CSS
+- Webpack
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Javascript DOM manipulation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link] (...come soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+* Have git installed. (For version control)
+* Have text editor installed. (VSCode, sublime, atom)
+* Have CSS,HTML & JS Linters installed and configured. For further information, visit [this link](https://github.com/microverseinc/linters-config/blob/master/README.md) to get better understanding on each linters used on this project.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone git@github.com:Juank628/to_do_list.git
+
+### Install
+
+Install this project with:
+
+npm install
+
+### Usage
+
+This page registers to-do activities. To register an activity:
+1. Type the activity's description.
+2. Press enter key.
+3. The activity will be added to the list and will be displayed.
+
+To remove a book from the list:
+1. Find the activity in the list.
+2. Click on the more button for the respective activity.
+3. Click on the delete button.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Juan Carlos Sanchez**
+
+- GitHub: [@Juank628](https://github.com/Juank628)
+- Twitter: [@juancadev81](https://twitter.com/juancadev81)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/juan-carlos-sanchez-zunino-271b5151)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Database connection**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Juank628/to_do_list/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
```

Comment:
- [ ] OPTIONAL
It's recommended as good practice that you add an MIT license, or any other similar license to your ReadMe for this project.